package.json

```
@@ -75,6 +75,18 @@
     "typescript": "^5.0.2"
   },
   "overrides": {
+    "@actions/cache": {
+      "semver": ">=6.3.1"
+    },
+    "@actions/tool-cache": {
+      "semver": ">=6.3.1"
+    },
+    "eslint-plugin-import": {
+      "semver": ">=6.3.1"
+    },
+    "eslint-plugin-jsx-a11y": {
+      "semver": ">=6.3.1"
+    },
```

Can you explain why this isn't sufficient? 

```suggestion
    "semver": "^7.5.4",
```

---

It's not possible to specify an override for a direct dependency.  Furthermore, I'd prefer to force upgrade dependencies to the same major version, to reduce the chances of breaking something.

---

I'd prefer to force upgrade dependencies to the same major version, to reduce the chances of breaking something.

---

thanks.